include/bio/record.hpp

```diff
@@ -239,17 +239,17 @@ struct record : seqan3::detail::transfer_template_args_onto_t<field_types_, std:
     BIO_RECORD_MEMBER(qual)
     BIO_RECORD_MEMBER(seq_qual)
     BIO_RECORD_MEMBER(offset)
-    BIO_RECORD_MEMBER(ref_id)
+    BIO_RECORD_MEMBER(rname)
     BIO_RECORD_MEMBER(ref_seq)
     BIO_RECORD_MEMBER(pos)
     BIO_RECORD_MEMBER(qname)
     BIO_RECORD_MEMBER(flag)
     BIO_RECORD_MEMBER(mapq)
     BIO_RECORD_MEMBER(cigar)
-    BIO_RECORD_MEMBER(next_ref_id)
-    BIO_RECORD_MEMBER(next_pos)
+    BIO_RECORD_MEMBER(rnext)
+    BIO_RECORD_MEMBER(pnext)
     BIO_RECORD_MEMBER(tlen)
-    BIO_RECORD_MEMBER(optionals)
+    BIO_RECORD_MEMBER(tags)
```

Comment:
```suggestion
    BIO_RECORD_MEMBER(sam_tags)
```
? Dunno. Maybe tags is OK, too.

Reply:
hm.. none of the other names is prefixed with its format. E.g. `tlen` also does not really tell you anything. 
On the other hand, tags is I think also made up from me :D 
Here is the respective paragraph form the SAM specs:
```
The alignment section: optional fields

All optional fields follow the TAG:TYPE:VALUE format where TAG is a two-character string that matches
/[A-Za-z][A-Za-z0-9]/. Within each alignment line, no TAG may appear more than once and the order
in which the optional fields appear is not significant. A TAG containing lowercase letters is reserved for end
users. In an optional field, TYPE is a single case-sensitive letter which defines the format of VALUE:
```
because it is called `TAG`:`VALUE` I guess I called it tags. I think it's more memorable than `optional_fields`? But I have no strong feelings on this. I am just against prefixing the format.

Reply:
I don't like `tag` because we have already other things called "tag" and it has a very specific meaning in C++ (it's always difficult when a term has one meaning in C++ and another in Bioinformatics).

I also decided to stay close to the format-terminology, so I would propose to just call the field "optionals".